exercises/grep/grep.js

```diff
@@ -0,0 +1,4 @@
+//
```

Comment:
```suggestion
#!/usr/bin/env node

// The above line is a shebang. On Unix-like operating systems, or environments, this will allow the script to be
// run by node, and thus turn this JavaScript file into an executable. If you don't have a Unix-like operating
// system or environment, for example Windows without WSL, you can use 
//
// node grep.js args
//
// Instead of "grep args".
// 
// Read more about shebangs here: https://en.wikipedia.org/wiki/Shebang_(Unix)
// 
```